steps/list-jira-tickets/0.0.1/step.yml

```diff
@@ -0,0 +1,69 @@
+title: JIRA tickets listing
+summary: |
+  Lists all JIRA tickets by comparing two branches.
+description: |
+  Lists all JIRA tickets by comparing two branches. It uses the JIRA ticket prefix to parse the commit messages. It will output either a list of ticket IDs or links if you add the URL path.
+website: https://bitbucket.org/silofit/bitrise-jira-tickets/src/master/
+source_code_url: https://bitbucket.org/silofit/bitrise-jira-tickets/src/master/
+support_url: https://bitbucket.org/silofit/bitrise-jira-tickets/src/master/
+published_at: 2021-08-17T15:09:12.991958-04:00
+source:
+  git: https://amarliac@bitbucket.org/silofit/bitrise-jira-tickets.git
```

Comment:
Why is the git URL different from the website and source code urls?

Reply:
Addressed